fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1013,6 +1009,15 @@ protected Builder() {
 
             }
 
+            /**
```

Comment:
Given the shared external builder, shouldn't we move the shared config class to an external file?

Reply:
We could. That would be an API breaking change, in that the fully qualified information for this class would change.

Reply:
I've moved it out into its own class with 34c1f70. There would need to be a reaction for any downstream project that was setting these values, though not an overly complicated one